### ISSUE TITLE
Improve Error Idenfication 

### ISSUE THREAD

**Issue by hannahackermans** [elmcip/elmcip#497]
### WCAG Guidelines

> **Success Criterion 3.3.1 Error Identification** 
> (Level A)
> If an [input error](https://www.w3.org/TR/WCAG21/#dfn-input-error) is automatically detected, the item that is in error is identified and the error is described to the user in text. 

> **Success Criterion 3.3.3 Error Suggestion** 
> (Level AA)
> If an [input error](https://www.w3.org/TR/WCAG21/#dfn-input-error) is automatically detected and suggestions for correction are known, then the suggestions are provided to the user, unless it would jeopardize the security or purpose of the content. 

### Current State of ELMCIP
ELMCIP has a error message at the top of the page indicating a mandatory field is not filled in or that an autocomplete field only has written text in it without linking to another record. The error identifications for missing information are useful (“[field title] is required”), but when one types something in an autocomplete field, the error message is simply "There are no entities matching "[what you wrote in the field]"" without stating in which field that error is. The fields where the error occurs only indicates the error. The error message also does not indicate that the error derives from not choosing an auto-complete option. 

![image](https://github.com/elmcip/elmcip/assets/18713442/8f8adfdb-5596-4686-ab8a-f7e06536c94e)

### Proposed changes to meet the success criteria 
1) I would like the error message to name the field in which the error occurs. 
2) As an error suggestion, indicate that the contributor must pick an option from the auto-complete menu. 
3) Also linking to issue #491 which addresses the use of color as indication. If there was a better way to indicate the actual field besides color, what would also make the error identification better.
